Clarify that this service does not support TensorFlow Lite (#39)
viam-modules/viam-mlmodelservice-triton: README.md

# Viam Triton ML Model Service

A Viam provided ML Model service backed by NVIDIA's [Triton Inference Server](https://developer.nvidia.com/triton-inference-server).

> Configure this ML Model service as a modular resource on your machine with a [Jetson board](https://docs.viam.com/build/configure/components/board/jetson/) to deploy TensorFlow (not TensorFlow Lite) or PyTorch ML models to your machine faster while consuming less computer power.

Reviewer: Triton actually supports several other model types: ONNX, OpenVino, TensorRT. I'm reluctant to try to document them all here, since I don't want us to be in the position of trying to document NVidia's stuff, and the exact set of supported model types potentially varies with version of the module (since that in turn uses a varying version of the Triton server). 

There is this page: https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/backend/docs/backend_platform_support_matrix.html.

But I can't find a versioned link. Also, it is important to note that not all model types are supported on all platforms. For instance, OpenVINO is not supported on Jetpack.

Author: ```suggestion
Configure this ML Model service as a modular resource on your machine with a [Jetson board](https://docs.viam.com/build/configure/components/board/jetson/) to deploy a [supported ML model](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/backend/docs/backend_platform_support_matrix.html) to your machine faster while consuming less computer power.
```

Good point. Instead of listing them here, I'll just link out -- at least then, users won't see outdated or incorrect information on our side. As long as something under the hood dictates support, it makes sense to link out IMO.

Author: Used a separate suggestion to implement this feedback along with another comment's suggestion, we can ignore this one.

> Configure this ML Model service as a modular resource on your machine with a [Jetson board](https://docs.viam.com/build/configure/components/board/jetson/) to deploy TensorFlow (not TensorFlow Lite) or PyTorch ML models to your machine faster while consuming less computer power.

Reviewer: You can use Triton server just fine on platforms that aren't Jetson (though that wasn't always true), or even that don't have an NVidia GPU (though it will only be able to use the CPU for inference).

Author: ```suggestion
Configure this ML Model service as a modular resource on your machine with a [Jetson board](https://docs.viam.com/build/configure/components/board/jetson/) or another supported machine to deploy [supported models](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/backend/docs/backend_platform_support_matrix.html) to your machine faster while consuming less computer power.
```

### Option 1: use a model from the Viam registry

You can download an ML model from Viam's registry. Head to
https://app.viam.com/registry?type=ML+Model to see the available models. Choose a model that uses either

> the TensorFlow or PyTorch model framework; this service does not support TensorFlow Lite. Once you've added this

Reviewer: Rather than having the list of supported models in two places, maybe link back to once place that has it.

> https://app.viam.com/registry?type=ML+Model to see the available models. Choose a model that uses either
> the TensorFlow or PyTorch model framework; this service does not support TensorFlow Lite. Once you've added this

Author: ```suggestion
https://app.viam.com/registry?type=ML+Model to see the available models. Choose a model that uses a [supported model framework](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/backend/docs/backend_platform_support_matrix.html). Once you've added this
```
While I would prefer to link higher up in the readme, I'm concerned that that might not translate well to the Viam app, so I opted to play it safe and link to the third-party doc instead.
